s4e-backend/src/main/java/pl/cyfronet/s4e/data/repository/query/QueryTime.java

```diff
@@ -59,6 +65,23 @@ private void queryBySensingTime(Map<String, Object> params,
         }
     }
 
+    private void applyLicense(Map<String, Object> params,
+                              List<Object> parameters,
+                              StringBuilder resultQuery) {
+        if (params.containsKey(ACCESS_TYPE) && params.get(ACCESS_TYPE).equals(Product.AccessType.EUMETSAT)) {
+            LocalDateTime now = LocalDateTime.now();
+            resultQuery.append(" AND ( ").append(getMetadataTime("sensing_time")).append(" <= ?");
```

Comment:
I'd remove the superfluous space after the opening paren, i.e. `" AND ( "` => `" AND ("`.
Same for below: `" OR  "` => `" OR "`, `" IN (?) )"` => `" IN (?))"`.